tests/glfw3.c

```diff
@@ -117,10 +124,12 @@ int main()
         glfwGetWindowPos(window, &x, &y); // stub
         glfwGetWindowSize(window, &w, &h);
         assert(w == 640 && h == 480);
+        assert(exited == 0);
```

Comment:
Does this assert fire if you won't include your patch?

Reply:
this assert is just in the incorrect line. thanks.

Reply:
And it fires, without this patch?  (i.e. if I add this assert locally I will see the test fail?)

Reply:
This assertion fires with `-sEXIT_RUNTIME=1`.
(I've used `--emrun`, which implies `-sEXIT_RUNTIME=1`)